web/src/pages/index.js

```diff
@@ -16,10 +16,13 @@ export default function IndexPage({ data }) {
             description:
                 'Etiam felis mauris, dapibus commodo ex a, vestibulum consequat neque. Vestibulum id auctor sapien. Vestibulum tempor tortor eget purus lobortis pretium.',
         },
+        hero: {
+            text: 'Donec eleifend faucibus finibus maximus',
+        },
     };
     return (
         <Layout>
-            <Hero image={heroImage} />
+            <Hero image={heroImage} data={content.hero} />
```

Comment:
Can we combine these to a single prop? Once sanity is integrated this will more than likely be one object that is passed to the component.

Reply:
@asasmith Yeah